pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/tablestate/TableStateUtils.java

```diff
@@ -38,6 +38,40 @@ public class TableStateUtils {
   private TableStateUtils() {
   }
 
+  /**
+   * Returns all online segments for a given table.
+   *
+   * @param helixManager instance of Helix manager
+   * @param tableNameWithType table for which we are obtaining ONLINE segments
+   *
+   * @return List of ONLINE segment names.
+   */
+  public static List<String> getTableOnlineSegments(HelixManager helixManager, String tableNameWithType) {
```

Comment:
Suggest rename to `getOnlineSegmentsForThisInstance()`

Reply:
done